lib/Runtime/Language/ModuleNamespace.cpp

```diff
@@ -111,4 +143,220 @@ namespace Js
         // keep a local copy of the module/
         unambiguousNonLocalExports->AddNew(propertyId, *nonLocalExportNameRecord);
     }
+
+    BOOL ModuleNamespace::HasProperty(PropertyId propertyId)
+    {
+        SimpleDictionaryPropertyDescriptor<BigPropertyIndex> propertyDescriptor;
+        const Js::PropertyRecord* propertyRecord = GetScriptContext()->GetThreadContext()->GetPropertyName(propertyId);
+        if (propertyRecord->IsSymbol())
+        {
+            return this->DynamicObject::HasProperty(propertyId);
+        }
```

Comment:
Is it possible to add properties to the namespace object? If not, this should always return false, right?